### ISSUE TITLE
getControlsContainerStyles & defaultControlsConfig props do not seem to work

### ISSUE THREAD

**Issue by amaidah** [FormidableLabs/nuka-carousel#663]
# Bugs and Questions

### Prerequisites

**Feel free to delete this section if you have checked off all of the following.**

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using? `"nuka-carousel": "4.6.1",`
- What version of React are you using? `"react": "^16.8.6",`
- What browser are you using? Chrome
- What machine are you on? Macbook Pro

### Describe the Problem

It's easier to show us than tell us what's going wrong with your code. Because of this, we ask that you do one of three things to help us reproduce the bug:

2. Create a sandbox, reproduce your issue in the code, and paste the link here.
This is forked from the official demo: https://codesandbox.io/s/quizzical-hermann-yv32g

Where I've added 
```
          getControlsContainerStyles={key => {
            console.log(key);
            switch (key) {
              case "CenterLeft": {
                return {
                  backgroundColor: "red",
                  height: "100%"
                };
              }
              case "CenterRight": {
                return {
                  backgroundColor: "blue"
                };
              }
              default: {
                return {};
              }
            }
          }}
          defaultControlsConfig={{
            nextButtonText: "Custom Next",
            prevButtonText: "Customn Prev",
            pagingDotsStyle: {
              fill: "red"
            }
          }}
```

**Expected behavior:** 
I expect that the styles of controls container to be updated to red for left and blue for right. I also expect the text to be overriden through `defaultControlsConfig`
**Actual behavior:** Nothing actually happens-- the props do not seem to be doing anything.

**Comment by nicco88** [comment]
Same thing here, Typescript is also giving me an error for these two props.

**Comment by nicco88** [comment]
@amaidah So I upgraded the version to 6.4.2 and it seems it works now, although I still get TS error.

**Comment by sarmeyer** [comment]
Yes, these props weren't introduced until 4.6.2, but that version has a dependency issue that is currently being worked on. What typescript error are you seeing?

**Comment by nicco88** [comment]
I see, thank you for your explanation. The TS error I get is this one:

`No overload matches this call.
  Overload 1 of 2, '(props: Readonly<CarouselProps>): Carousel', gave the following error.
    Type '{ children: Element[]; wrapAround: true; autoplay: true; pauseOnHover: true; autoplayInterval: number; speed: number; easing: string; renderCenterLeftControls: null; renderCenterRightControls: null; beforeSlide: (i: number) => void; defaultControlsConfig: { ...; }; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Carousel> & Readonly<CarouselProps> & Readonly<...>'.
      Property 'defaultControlsConfig' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Carousel> & Readonly<CarouselProps> & Readonly<...>'.
  Overload 2 of 2, '(props: CarouselProps, context?: any): Carousel', gave the following error.
    Type '{ children: Element[]; wrapAround: true; autoplay: true; pauseOnHover: true; autoplayInterval: number; speed: number; easing: string; renderCenterLeftControls: null; renderCenterRightControls: null; beforeSlide: (i: number) => void; defaultControlsConfig: { ...; }; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Carousel> & Readonly<CarouselProps> & Readonly<...>'.
      Property 'defaultControlsConfig' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Carousel> & Readonly<CarouselProps> & Readonly<...>'.ts(2769)`

**Comment by sarmeyer** [comment]
fixed in #669  - will be released later today
